### PR TITLE
Dependents 81049: Add V2 redirect

### PR DESCRIPTION
**Pull request by Midge-dev**
## Summary

- Adds entry point for 686c v2

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/81049

## Testing done

Visited /view-change-dependents/add-remove-form-21-686c-v2 and saw form application load

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by tblackwe — DISMISSED** [review]

File: src/platform/forms/constants.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const VA_FORM_IDS = Object.freeze({
   FORM_21_4142: '21-4142',
   FORM_21_526EZ: '21-526EZ',
   FORM_21_686C: '686C-674',
+  FORM_21_686CV2: '686C-674V2',
```
Comment (tblackwe) [inline src/platform/forms/constants.js:27]: does it matter that this is `686C-674V2` as opposed to `686C-674-V2` ?

**Review by tblackwe — DISMISSED** [review]

File: src/platform/forms/constants.js (hunk 1 above)
Comment (Midge-dev) [reply src/platform/forms/constants.js]: I pushed a change for this. Tested everything and it's working as expected

**Review by tblackwe — APPROVED** [review]
when will my approval be enough for you Sean....

**Review by micahaspyr — APPROVED** [review]
👍

**Review by powellkerry — APPROVED** [review]

**Pull request merged**